ui/imports/shared/views/chat/LinksMessageView.qml

```diff
@@ -113,8 +114,7 @@ Flow {
         delegate: LinkPreviewCardDelegate {
             id: delegate
 
-            utilsStore: root.utilsStore
-
+            cbGetEmojiHash: root.cbGetEmojiHash
```

Comment:
it seems that every instance created by the repeater takes the same `cbGetEmojiHash` and calls it for the same pub key. Hopefully,we could take the emoji hash here directly.

Reply:
Probably `LiksMessagePreview` can just take the emoji hash directly, not via a function.

Reply:
Ok, I missed the fact that it's dependent on the links model content. Then the cleanest way is probably to provide that hash via model in `link_preview_model.nim`.